packages/convergence/README.md

```diff
@@ -39,3 +39,336 @@ that condition has been met.
+or `20ms` (whichever is greater).
+
+``` javascript
+converge
+  // will be given any remaining timeout
+  .always(() => total === 5)
+
+converge
+  // will use the 500ms timeout instead
+  .always(() => total === 5, 500)
+  .once(() => total === 10)
+
+new Convergence(2000)
+  // defaults to 200ms
+  .always(() => total === 5)
+  // this counts as part of the stack
+  .do(() => console.log(total))
+```
+
+**`converge.do(exec)`**
+
+This method is useful when you need to execute something after a
+convergence in the stack, but before other convergences are ran.
+This can help with debugging in between convergences and also
+allows you to run side effects between convergences.
+
+``` javascript
+converge
+  .once(() => total === 5)
+  // executes after the total is equal to 5
+  .do(() => total *= 100)
+  // starts converging after the total has been multiplied
+  .always(() => total === 500)
+```
+
+Just like `convergeOn` resolves with the return value of the
+assertion, functions in a `Convergence` stack curry their return value
+to the next function in the stack.
+
+``` javascript
+converge
+  // returns the element when it exists in the DOM
+  .once(() => {
+    let $el = $('[data-test-element]');
+    expect($el).to.exist;
+    return $el;
+  })
+  // clicks the element and returns it for chaining
+  .do(($el) => {
+    $el.get(0).click();
+    return $el;
+  })
+  // converges when the element is disabled for the remaining timeout
+  .always(($el) => {
+    expect($el).to.have.prop('disabled', true)
+  })
+```
+
+**`converge.run()`**
+
+In order to actually run a `Convergence` instance, you must call the
+`.run()` method. This method **does not return another instance**, instead
+it returns a promise that resolves when all assertions in the stack
+have converged. The returned promise will keep track of the current
+timeout and ensure that all convergences in the stack converge within
+that period.
+
+The promise returned from `.run()` will _always_ resolve with a stats
+object. This stats object is similar to the stats object returned by
+`convergeOn`, but missing `always`, and additionally includes a `stack`
+property. This `stack` property is simply an array of stats objects
+returned for the various convergences in the stack. `.do()` blocks
+also produce a small subset of the `stats` object.
+
+``` javascript
+converge.run((stats) => {
+  stats.start    // start time of the convergences
+  stats.end      // end time of the convergences
+  stats.elapsed  // time taken to converge on the entire stack
+  stats.runs     // number of times functions in the stack ran
+  stats.timeout  // this Convergence instance's timeout
+  stats.value    // value returned from the last function in the stack
+  stats.stack    // an array of stats objects from the stack
+});
+```
+
+Because `.run()` returns a promise, and `Convergence` is immutable,
+you can call `.run()` multiple times on the same convergence instance.
```

Comment:
I feel like this README ends kind of abruptly. Maybe some section to tie it all up? Or in the future we could link to other `@bigtest/*` packages that use `@bigtest/convergence`. Thoughts?